internal/util/plugins/init.go

```diff
@@ -0,0 +1,99 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// TODO: rewrite this when plugins phase 2 is implemented.
+package plugins
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"sigs.k8s.io/kubebuilder/pkg/model/config"
+
+	"github.com/operator-framework/operator-sdk/internal/util/projutil"
+)
+
+// RunInit modifies the project scaffolded by kubebuilder's Init plugin.
+func RunInit(cfg *config.Config) error {
```

Comment:
This isn't actually "running" init, it is updating the Makefile
```suggestion
func UpdateMakefile(cfg *config.Config) error {
```

Reply:
done